deploy/docs/Non_Helm_Installation.md

```diff
@@ -6,9 +6,7 @@ This document has instructions for setting up collection with Fluentd, FluentBit
 
 - [Before you start](#before-you-start) 
 - [Create Sumo Fields, a collector, and deploy Fluentd](#create-sumo-fields-a-collector-and-deploy-fluentd) 
-  - [Automatic Source Creation and Setup Script](#automatic-source-creation-and-setup-script) 
-    - [Parameters](#parameters) 
-    - [Environment variables](#environment-variables) 
+  - [Automatic Source Creation and Setup YAML](#automatic-source-creation-and-setup-yaml) 
   - [Manual Source Creation and Setup](#manual-source-creation-and-setup) 
     - [Create a Hosted Collector and an HTTP Source](#create-a-hosted-collector-and-an-http-source) 
     - [Deploy Fluentd](#deploy-fluentd) 
```

Comment:
Since the automatic setup with terraform no longer deploys Fluentd, we should move the step `- [Deploy Fluentd](#deploy-fluentd)` up a level in the TOC to the same level as `Automatic Source ...` and `Manual Source ...`

Reply:
In fact, I take that back - can we split the top level step from `- [Create Sumo Fields, a collector, and deploy Fluentd](#create-sumo-fields-a-collector-and-deploy-fluentd)` to just

```
- Create Sumo Fields and a collector
- Deploy Fluentd
- Configure Prometheus
...
```

Reply:
Sorry for bombarding with comments, but also a side question - can we rename `Configure Prometheus` to `Deploy Prometheus`, just for consistency with the other `Deploy ___` steps that we have?

Reply:
good points, changes coming